pkgs/by-name/openfire/package.nix

```
@@ -7,13 +7,13 @@
 }:
 maven.buildMavenPackage rec {
   pname = "openfire";
-  version = "4.9.0";
+  version = "4.9.2";
 
   src = fetchFromGitHub {
     owner = "igniterealtime";
     repo = "Openfire";
     rev = "v${version}";
```

This was introduced a while ago to avoid conflicts with branches that share the same name as the tag. Although this package's source repo doesn't have such branches, IMO it's a good pattern to adopt when working with tags.

```suggestion
    tag = "v${version}";
```

---

Yes. It's the better pattern.